force_wfmanager/left_side_pane/variable_names_registry.py

```diff
@@ -0,0 +1,66 @@
+from traits.api import HasStrictTraits, List, Instance, on_trait_change
+
+from force_bdss.api import Identifier
+
+from .mco_parameter_model_view import MCOParameterModelView
+from .data_source_model_view import DataSourceModelView
+from .kpi_calculator_model_view import KPICalculatorModelView
+
+
+class VariableNamesRegistry(HasStrictTraits):
```

Comment:
And thinking about it, it does make sense. Because the `available_variables` attribute is only there to drive the possible values of the comboboxes, which is totally view related.